src/styles/createPalette.js

```diff
@@ -77,21 +77,48 @@ export const dark = {
 
 export default function createPalette(palette: Object) {
   const {
-    primary = indigo,
-    secondary = pink,
-    error = red,
+    primary = {
+      light: indigo[300],
```

Comment:
Question, why using 300 and not a lighter version of `indigo[500]`?
(we could just remove this line so the default logic quicks in)

Reply:
Bacuase it's the "correct" color according to the MD colors. But we can approximate if you're happy with that.

Reply:
Sounds good then :)

Reply:
Sorry, which?

Reply:
@mbrookes Your argument sounds good. I thought MCW was doing the opposite.